codes/classical/q-ary_digits/skew_cyclic.yml

```diff
@@ -0,0 +1,119 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#},
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are 
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: skew_cyclic
```

Comment:
Is it true that any quasi-cyclic code is also skew-cyclic? If so, then I would argue we should combine the two entries into one because having two identical code families seems confusing.

Reply:
Reading more closely, the paper only proves that for every skew-cyclic code there is an equivalent quasi-cyclic code, they do not consider the other direction. For now I have labeled quasi-cyclic as a parent, would it make more sense as a cousin?

Reply:
Parent works. Thanks for looking into it.